Source/WebCore/page/RemoteFrameView.h

```diff
@@ -66,6 +66,8 @@ class RemoteFrameView final : public AbstractFrameView {
     void updateLayerPositionsAfterScrolling() final;
     void updateCompositingLayersAfterScrolling() final;
 
+    void setFrameRect(const IntRect&) final;
```

Comment:
What is the coordinate system of the rect?

Reply:
I don't know.